src/frontend/components/StudentLiveWrapper/index.tsx

```diff
@@ -46,6 +49,23 @@ export const StudentLiveWrapper: React.FC<StudentLiveWrapperProps> = ({
   );
   const [showPanelTrigger, setShowPanelTrigger] = useState(true);
 
+  useEffect(() => {
+    if (isParticipantOnstage) {
+      const interval = window.setInterval(() => {
+        const attendance = {
+          [Date.now()]: {
+            onStage: true,
+          },
+        };
+        pushAttendance(attendance, intl.locale, getOrInitAnonymousId());
+      }, PUSH_ATTENDANCE_DELAY);
+      return () => {
+        // Stop tracking attendances as we unmount
+        window.clearInterval(interval);
+      };
+    }
+  }, [isParticipantOnstage]);
```

Comment:
you may add `intl` in the dependency array as well

Reply:
should be `intl.locale` in fact.